src/main/java/com/stripe/model/Balance.java

```diff
@@ -99,19 +99,19 @@ public static Balance retrieve(Map<String, Object> params, RequestOptions option
     String currency;
 
     @SerializedName("source_types")
-    MoneySourceTypes sourceTypes;
-  }
+    SourceTypes sourceTypes;
 
-  @Getter
-  @Setter
-  @EqualsAndHashCode(callSuper = false)
-  public static class MoneySourceTypes extends StripeObject {
-    /** Amount for bank account. */
-    @SerializedName("bank_account")
-    Long bankAccount;
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class SourceTypes extends StripeObject {
```

Comment:
Compatible with 7.x, 3rd level nesting

Reply:
not sure what this comment means?

Reply:
I meant that `SourceTypes` class now is nested inside `Money`, and inside `Balance` just as how 7.x has.